### PR TITLE
[ND] Hide the aircraft indicator in PLAN mode while ADIRS are not aligned

### LINKED ISSUE #186: ND PLAN Mode shows aircraft when ADIRS off
Hi,

I found that the PLAN Mode of the ND haven't a no ADIRS mode, if a make FPLN in the MCDU without ADIRS on (wich is in theory impossible) the PLAN page is showing up normally, so there is missing a "MAP NOT AVAIL" in this mode. 

image

Should be this :
image

### PR DESCRIPTION
**Pull request by lousybyte**
Fixes #186, #959

**Summary of Changes**

Prevent the aircraft indicator from showing in the PLAN mode screen while ADIRS are not aligned.

**Screenshots (if necessary)**

alt text

**References**

[https://www.youtube.com/watch?v=MywqXeeoYos&feature=youtu.be&t=745](https://www.youtube.com/watch?v=MywqXeeoYos&feature=youtu.be&t=745)

### CODE REVIEW

**Review by DoToAdventures — APPROVED** [review]
LGTM! Ready for testing

**Comment by ImenesFBW** [comment]
Name: Imenes#8739
Date of testing: 3/10/2020
Version of the sim: 1930
PR Tested: #1038

Testing technique: Try-out
Steps made to test the Pull Request:

1. Loaded the plane C&D, 
2. Waited for ADIRS to align.

Overall Rating: Good
Conclusions of the testing: Working as documented, LGTM! Thank you for your work!

Pics from the test:

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -3,13 +3,6 @@
 <!-- Always use "1." at the start instead of "2. " or "X. " as GitHub will auto renumber everything. -->
 <!-- Use the following format below -->
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
-## Changes from 2020/10 to 2020/11
-1. [ND] Change ND Sizing - @nathaninnes (Nathan Innes)
-
-1. [GPWS] Added Pull Up, Sink Rate, Dont sink GPWS with Default Sounds & Added (Too low) terrain, flaps and gear without Sounds (Future PR by Boris) - @oliverpope03 (Oliver Pope)
-
-1. [Sounds] Added new sounds for fuel pumps, flaps, ground roll and rattles, touchdowns, and wind - @hotshotp (Boris)
-
```
Comment (NathanInnes) [inline .github/CHANGELOG.md:12]: Did you mean to change the order here?

**Review by NathanInnes — COMMENTED** [review]
Did you mean to change the order here??

File: .github/CHANGELOG.md (hunk 1 above)
Comment (lousybyte) [reply .github/CHANGELOG.md:12]: Yes the current month was added to the bottom, since everyone kept adding changes there.

**Pull request merged**